client/src/components/core/Modals/DeleteBoardModal 2.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import styles from "./Modals.module.css";
+
+const DeleteBoardModal = ({ isOpen, onClose, onDelete }) => {
+	if (!isOpen) return null;
+
+	return (
+		<div className={styles.modalOverlay}>
```

Comment:
It is always good practice to use native elements. For accessibility purposes, we always prefer using <dialog /> elements for these type of things.. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dialog I created this guy: https://github.com/Erikang20/Kanban-task-management-web-app/blob/main/client/src/components/core/Modals/Dialog.tsx with the intention to be reused and render conditionally with the information we needed. We don't have to use this component at all, but I thought I mention it :)

Reply:
This is very helpful! I didn't actually create the modal for this. It looks like this is another duplicate file. But now that I've seen this component, I think it will be very useful for creating our other modals.